README.md

```diff
@@ -225,6 +225,25 @@ options:
 
 ```
 
+## Assumptions
+
+* **This is a cryptographic protocol conducted via a REST API.**  In theory, it
+  is agnostic to the underlying transport.  In practice, all parties
+  communicate with the server via HTTP over Tor.
+
+* **Message expiry/deletion will occur on a fuzzy interval.** The protocol will expire
```

Comment:
I think this would let us delete the "Message Retention" section under "Limitations and Discussion".

Reply:
Good point - pulled in some of that info to this bullet point and deleted that section.